internal/virtualcircuit/create.go

```diff
@@ -0,0 +1,172 @@
+			if err != nil {
+				return fmt.Errorf("could not create Virtual Circuit: %w", err)
+			}
+
+			data := make([][]string, 1)
+
+			if vrfFlag != "" {
+				data[0] = []string{vc.VrfVirtualCircuit.GetId(), vc.VrfVirtualCircuit.GetName(), string(vc.VrfVirtualCircuit.GetType()), strconv.Itoa(int(vc.VrfVirtualCircuit.GetSpeed())), vc.VrfVirtualCircuit.GetCreatedAt().String()}
+			} else {
+				data[0] = []string{vc.VlanVirtualCircuit.GetId(), vc.VlanVirtualCircuit.GetName(), string(vc.VlanVirtualCircuit.GetType()), strconv.Itoa(int(vc.VlanVirtualCircuit.GetSpeed())), vc.VlanVirtualCircuit.GetCreatedAt().String()}
+			}
+
+			header := []string{"ID", "Name", "Type", "Speed", "Created"}
+
+			return c.Out.Output(vc.VlanVirtualCircuit, header, &data)
+		},
+	}
+	// Virtual Circuit Params
+	createVirtualCircuitCmd.Flags().StringVarP(&connectionID, "connId", "i", "", "Specify the UUID of the interconnection.")
+	createVirtualCircuitCmd.Flags().StringVarP(&portID, "portId", "p", "", "Specify the UUID of the port.")
+
+	// Common params for both VlanVirtualCircuit and VrfVirtualCircit
+	createVirtualCircuitCmd.Flags().IntVarP(&nnVlan, "vlan", "V", 0, "Adds or updates vlan  Must be between 2 and 4094")
+	createVirtualCircuitCmd.Flags().IntVarP(&speed, "speed", "s", 0, "bps speed or string (e.g. 52 - '52m' or '100g' or '4 gbps')")
+	createVirtualCircuitCmd.Flags().StringSliceVarP(&tags, "tags", "t", []string{}, "Return only the specified tags.")
+	createVirtualCircuitCmd.Flags().StringVarP(&name, "name", "n", "", "Name of the Virtual Circuit")
+	createVirtualCircuitCmd.Flags().StringVarP(&description, "description", "d", "", "Description for a Virtual Circuit")
+	createVirtualCircuitCmd.Flags().StringVarP(&projectID, "projectId", "P", "", "The project's UUID. This flag is required, unless specified in the config created by metal init or set as METAL_PROJECT_ID environment variable.")
+
+	//VlanVirtualCircuit input params
+	createVirtualCircuitCmd.Flags().StringVarP(&vnid, "vnid", "v", "", "Specify the UUID  of the VLAN.")
+
+	// VrfVirtualCircuit input params
+	createVirtualCircuitCmd.Flags().StringVarP(&md5, "md5", "M", "", "The plaintext BGP peering password shared by neighbors as an MD5 checksum")
+	createVirtualCircuitCmd.Flags().IntVarP(&peerAsn, "peerAsn", "a", 0, "The peer ASN that will be used with the VRF on the Virtual Circuit.")
+	createVirtualCircuitCmd.Flags().StringVarP(&vrf, "vrf", "R", "", "The UUID of the VRF that will be associated with the Virtual Circuit.")
+	createVirtualCircuitCmd.Flags().StringVarP(&subnet, "subnet", "S", "", "The /30 or /31 subnet of one of the VRF IP Blocks that will be used with the VRF for the Virtual Circuit. ")
+	createVirtualCircuitCmd.Flags().StringVarP(&customerIP, "customerIP", "c", "", "An IP address from the subnet that will be used on the Customer side")
+	createVirtualCircuitCmd.Flags().StringVarP(&metalIP, "metalIP", "m", "", "An IP address from the subnet that will be used on the Metal side. ")
```

Comment:
These should be `--customer-ip` and `--metal-ip` to align with the format we use for command-line flags.  IMO we can leave out the short flag here